controllers/rabbitmqcluster_controller_test.go

```diff
@@ -534,34 +534,13 @@ var _ = Describe("RabbitmqClusterController", func() {
 				}, 3).Should(HaveKeyWithValue(annotationKey, annotationValue))
 			})
 
-			It("updates annotations for config maps", func() {
-				Eventually(func() map[string]string {
-					roleBinding, err := clientSet.CoreV1().ConfigMaps(cluster.Namespace).Get(ctx, cluster.ChildResourceName("server-conf"), metav1.GetOptions{})
-					Expect(err).NotTo(HaveOccurred())
-					return roleBinding.Annotations
-				}, 3).Should(HaveKeyWithValue(annotationKey, annotationValue))
-
-				Eventually(func() map[string]string {
-					roleBinding, err := clientSet.CoreV1().ConfigMaps(cluster.Namespace).Get(ctx, cluster.ChildResourceName("plugins-conf"), metav1.GetOptions{})
-					Expect(err).NotTo(HaveOccurred())
-					return roleBinding.Annotations
-				}, 3).Should(HaveKeyWithValue(annotationKey, annotationValue))
-			})
-
-			It("updates annotations for config maps", func() {
-				Eventually(func() map[string]string {
-					roleBinding, err := clientSet.CoreV1().ConfigMaps(cluster.Namespace).Get(ctx, cluster.ChildResourceName("server-conf"), metav1.GetOptions{})
-					Expect(err).NotTo(HaveOccurred())
-					return roleBinding.Annotations
-				}, 3).Should(HaveKeyWithValue(annotationKey, annotationValue))
-
+			It("updates annotations for plugins config map", func() {
```

Comment:
I'm wondering whether we should not update plugins config map either since a label gets added to that config map and `rabbitmq plugins-set` get executed on every pod of the StatefulSet for every Flux reconciliation.

Reply:
I think you are right. I am really not a fan of having special behaviors for the configmaps, but I updated it. Nice catch David!